doggystyle.css

```diff
@@ -0,0 +1,186 @@
+/*adoptadog stylesheet*/
+* {
+    box-sizing: border-box;
+}
+
+
+header {
+  width: 100%
+  border: none;
+  margin: none;
+  background-color: DarkCyan;
+  padding: 2%;
+}
+
+header {
+  display: flex;
+  flex-flow: row;
+}
+
+main {
+  margin-top: 0;
+  margin-right: 10%;
+  margin- bottom: 5%;
+  margin-left: 10%;
+}
+
+h3 {
+  font-size: 20px;
+  font-family: Gill sans, sans-serif;
+  color: chartreuse;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.checkouticon {
+  margin: 2%;
+}
+
+.checkbutton {
+  background-color: chartreuse;
+  color: white;
+  padding: 5px 30px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 70%;
+}
+
+nav {
+  width: 100%;
+  margin: 0 0 1%;
+  padding: .5%;
+  background-color: pink;
+}
+
+li {
+  display: inline;
+  padding: 1%;
+  font-size: 82%;
+}
+
+.topnav {
+  color: DarkCyan;
+  text-decoration: none;
+  fontfamily: Gill sans, sans-serif;
+  font-weight: bold;
+}
+
+h1 {
+  color: DarkCyan;
+}
+
+h2 {
+  color: DarkCyan;
+  }
+
+.main-blog-para {
+  display: flex;
+  flex-direction: row;
+}
+
+.intro-para {
+  padding: 2%;
+}
+
+p {
+  font-family: sans-serif;
+  padding-top: 0;
+  padding-right: 0;
+  padding-bottom: 1%;
+  padding-left: 1%;
+}
+
+section {
+  display: flex;
+  flex-direction: row;
+}
+
+.dogbox {
+  width: 15%;
+  height: 30%;
```

Comment:
I'd stick with a px value for height. Height is somewhat strange when using percents.